Explorer/Assets/DCL/Multiplayer/Movement/Settings/Rules/SendRuleBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DCL.CharacterMotion.Components;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Movement.Settings
+{
+    public abstract class SendRuleBase : ScriptableObject
+    {
+        public Color color = Color.white;
+        public float MinTimeDelta = 0;
+        public virtual string Message { get; set; }
+
+        public abstract bool IsSendConditionMet(float t,
```

Comment:
I believe the arguments you pass here should not be mutable so it's better to pass them by `in` to clarify your intentions